.env.example

```diff
@@ -1,9 +1,9 @@
 # FRONTEND
-PORT=3100
+CLIENT_PORT=3100
 VITE_METABASE_INSTANCE_URL="http://localhost:3000"
 VITE_AUTH_PROVIDER_URI="http://localhost:9090/sso/metabase"
 
 # BACKEND
-BACKEND_PORT=9090
-METABASE_INSTANCE_URL= "http://localhost:3000"
+AUTH_PROVIDER_PORT=9090
+METABASE_INSTANCE_URL="http://localhost:3000"
 METABASE_JWT_SHARED_SECRET="TODO"
```

Comment:
Fyi @albertoperdomo 
I will update documentation as soon as this PR is merged

Reply:
https://github.com/metabase/metabase.github.io/pull/4438
https://github.com/metabase/metabase/pull/54551
https://github.com/metabase/metabase/pull/54552
https://github.com/metabase/metabase/pull/54553

Reply:
@sanex3339 the first pr is against the actual website (see Vamsi comment)
the other are closed

Reply:
Updated links

Reply:
> the first pr is against the actual website
This is fine, I'm going to do the manual update as mentioned in notion